service/controller/resource/monitoring/remotewriteagentconfigsecret/doc.go

```diff
@@ -0,0 +1,4 @@
+package remotewriteagentconfigsecret
+
+// This packages creates the remote write agent secret config
+// to be used as values by the prometheus agent app to configure the remote write feature.
```

Comment:
If I got this correctly in this package we are wrapping the `RemoteWrite` config in a secret which will then be used by prometheus-operator to configure the agent. Is this right ?

Reply:
We could eventually fill this with the remotewrite CR as well